Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -450,18 +451,18 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
             // Dark blurred background.
             blurView.autoPinEdgesToSuperviewEdges()
 
-            localVideoView.autoPinEdge(toSuperviewEdge:.right, withInset:videoPreviewHMargin)
+            localVideoView.autoPinTrailingToSuperView(withMargin: videoPreviewHMargin)
             localVideoView.autoPinEdge(toSuperviewEdge:.top, withInset:topMargin)
             let localVideoSize = ScaleFromIPhone5To7Plus(80, 100)
             localVideoView.autoSetDimension(.width, toSize:localVideoSize)
             localVideoView.autoSetDimension(.height, toSize:localVideoSize)
 
             contactNameLabel.autoPinEdge(toSuperviewEdge:.top, withInset:topMargin)
-            contactNameLabel.autoPinEdge(toSuperviewEdge:.left, withInset:contactHMargin)
```

Comment:
Okay, I've widened this up a bit.  I made a few small changes like this as I converted things - the idea was by honoring layoutMargins instead of our arbitrary margin/spacing values, we could feel a little bit more idiomatic to iOS.  But I agree, in this case, it's too much.